sdk/spring/azure-spring-boot/src/test/java/com/microsoft/azure/spring/autoconfigure/aad/MicrosoftGraphConstants.java

```diff
@@ -0,0 +1,121 @@
+        + "            \"classification\": null,\n"
+        + "            \"createdDateTime\": \"2017-08-09T13:45:03Z\",\n"
+        + "            \"creationOptions\": [],\n"
+        + "            \"description\": \"this is group2\",\n"
+        + "            \"displayName\": \"group2\",\n"
+        + "            \"groupTypes\": [],\n"
+        + "            \"mail\": null,\n"
+        + "            \"mailEnabled\": false,\n"
+        + "            \"mailNickname\": \"somethingelse\",\n"
+        + "            \"onPremisesLastSyncDateTime\": null,\n"
+        + "            \"onPremisesSecurityIdentifier\": null,\n"
+        + "            \"onPremisesSyncEnabled\": null,\n"
+        + "            \"preferredDataLocation\": null,\n"
+        + "            \"proxyAddresses\": [],\n"
+        + "            \"renewedDateTime\": \"2017-08-09T13:45:03Z\",\n"
+        + "            \"resourceBehaviorOptions\": [],\n"
+        + "            \"resourceProvisioningOptions\": [],\n"
+        + "            \"securityEnabled\": true,\n"
+        + "            \"visibility\": null,\n"
+        + "            \"onPremisesProvisioningErrors\": []\n"
+        + "        },\n"
+        + "        {\n"
+        + "            \"@odata.type\": \"#microsoft.graph.group\",\n"
+        + "            \"id\": \"12345678-86a4-4237-aeb0-60bad29c1de0\",\n"
+        + "            \"deletedDateTime\": null,\n"
+        + "            \"classification\": null,\n"
+        + "            \"createdDateTime\": \"2017-08-09T05:41:43Z\",\n"
+        + "            \"creationOptions\": [],\n"
+        + "            \"description\": \"this is group3\",\n"
+        + "            \"displayName\": \"group3\",\n"
+        + "            \"groupTypes\": [],\n"
+        + "            \"mail\": null,\n"
+        + "            \"mailEnabled\": false,\n"
+        + "            \"mailNickname\": \"somethingelse\",\n"
+        + "            \"onPremisesLastSyncDateTime\": null,\n"
+        + "            \"onPremisesSecurityIdentifier\": null,\n"
+        + "            \"onPremisesSyncEnabled\": null,\n"
+        + "            \"preferredDataLocation\": null,\n"
+        + "            \"proxyAddresses\": [],\n"
+        + "            \"renewedDateTime\": \"2017-08-09T05:41:43Z\",\n"
+        + "            \"resourceBehaviorOptions\": [],\n"
+        + "            \"resourceProvisioningOptions\": [],\n"
+        + "            \"securityEnabled\": true,\n"
+        + "            \"visibility\": null,\n"
+        + "            \"onPremisesProvisioningErrors\": []\n"
+        + "        }"
+        + "],\n"
+        + "    \"odata.nextLink\": \"directoryObjects/$/Microsoft.DirectoryServices.User/"
+        + "12345678-2898-434a-a370-8ec974c2fb57/memberOf?$skiptoken=X'4453707407000100000000"
+        + "00000000100000009D29CBA7B45D854A84FF7F9B636BD9DC000000000000000000000017312E322E3"
+        + "834302E3131333535362E312E342E3233333100000000'\"\n"
+        + "}";
```

Comment:
This json can be read from a file in resources instead of adding this long json with escaping. It's easier to update too.

Reply:
done.